### PR TITLE
Add .pre-commit-hooks.yaml

### PR DESCRIPTION
**Pull request by matilda-me**
Pandas project asked if I could add sphinx-lint as a pre-commit hook, is it OK if it means adding a file here?

I also had to allow multiple files to be passed as arguments.

### CODE REVIEW

File: .pre-commit-hooks.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- id: sphinx-lint
+  name: Sphinx lint
+  description: 'Searches for common typos in sphinx-flavord rst files.'
```
Comment (Jean-Abou-Samra) [inline .pre-commit-hooks.yaml:3]:
```suggestion
  description: 'Searches for common typos in sphinx-flavored rst files.'
```

**Comment by JulienPalard** [comment]
I don't really get the whole ".pre-commit-hooks.yaml", I bet I'll have to take a deeper look at it!

LGTM!

**Comment by JulienPalard** [comment]
@matilda-me I bet you need a release to be able to start using the hook?

I'll try to do one, soon™.

**Comment by matilda-me** [comment]
> @matilda-me I bet you need a release to be able to start using the hook?

Technically it work with `rev: main` but the PR will probably not be accepted like this. I'll wait for a release.

Technically it just need a tag, not a release.

**Comment by JulienPalard** [comment]
I just released sphinx-lint v0.2

**Pull request merged**